content/roadmap.md

```diff
@@ -4,6 +4,20 @@ title: Roadmap
 
 The following is only a selection of some of the major features we plan to implement in the near future (6-12 months). To get a more complete overview of planned features and current work, see the issue trackers for the various repositories, for example, the [Jaeger backend](https://github.com/jaegertracing/jaeger/issues/).
 
+## Integration with OpenTelemetry collector
+
+[OpenTelemetry collector](https://opentelemetry.io/docs/collector/about) is a vendor-agnostic service for receiving, processing and exporting telemetry data. We have decided to deprecate the Jaeger backed components (agent, collector, ingester, all-in-one) and migrate its functionality to an implementation based on OpenTelemetry collector which has several benefits:
+
+* forward compatibility with OpenTelemetry native data model
+* tail-based sampling
+* attribute processors
+* [standardized collection pipeline](https://opentelemetry.io/)
+* less code to maintain
```

Comment:
Rather than less code to maintain, might be better to say something like "leverage a larger community"? i.e. something more positive than reducing code maintenance :)